pages/pipelines/controlling_concurrency.md

```diff
@@ -42,6 +42,9 @@ For example, if you have two pipelines that each deploy to a different target bu
 
 Concurrency groups guarantee that jobs will be run in the order that they were created in. Jobs inherit the creation time of their parent. Parents of jobs can be either a build or a pipeline upload job. As pipeline uploads add more jobs to the build after it has started, the jobs that they add will inherit the creation time of the pipeline upload rather than the build.
 
+> 🚧 `concurrency_group` and `block` / `input` steps
+> Note that both [block](/docs/pipelines/block-step) and [input](/docs/pipelines/input-step) steps cause the steps to be uploaded and scheduled at the same time, which breaks concurrency groups. These two steps block jobs being added to the concurrency group, but do not adjust ordering once unblocked. The concurrency won't be added to the queue until the block step is unblocked, and once it does, the timestamp will be from the pipeline upload step.
```

Comment:
@pzeballos , I've revised this sentence a tad. Please let me know if my revisions are correct. Otherwise, please clarify what's incorrect.

Also, I have a few more queries about the existing text additions in this PR:

> but do not adjust ordering once unblocked.

Do you mean that jobs will still run in the order they were created in, or does using `block` and `input` steps break this ordering too?

> The concurrency won't be added to the queue

Do you mean the _concurrency group_? If not, what do you mean by 'concurrency' on its own?

> until the block step is unblocked

Is this _only_ a `block` step, or does this cover both `block` and `input` steps?

Sorry to be pedantic here, but I'm aiming to make the docs as unambiguous as possible.

Reply:
Thanks @gilesgas!

> Do you mean that jobs will still run in the order they were created in, or does using block and input steps break this ordering too?
They will run in the order they were created.

> Do you mean the concurrency group? If not, what do you mean by 'concurrency' on its own?
The concurrency group

> Is this only a block step, or does this cover both block and input steps?
Both :)

Reply:
Wonderful! Thanks for these updates Paula :-)